distutils/bcppcompiler.py

```diff
@@ -294,8 +294,7 @@ def link(  # noqa: C901
                     ld_args.append(libfile)
 
             # some default libraries
-            ld_args.append('import32')
-            ld_args.append('cw32mt')
+            ld_args.extend(('import32', 'cw32mt'))
```

Comment:
I slightly prefer the readability of the other syntax, but this form is okay, and apparently preferred by refurb.

Reply:
Agreed, I'm not always happy with refurb suggestions.